tests/src/setupFiles/extended-hooks.ts

```diff
@@ -23,11 +23,7 @@ import * as path from 'node:path';
 
 afterEach(async (context: RunnerTestContext) => {
   context.onTestFailed(async () => {
-    const normalizedFilePath = context.task.name
-      .replace(/'/g, '')
-      .replace(/\//g, '_')
-      .replace(/:/g, '_')
-      .replace(/ /g, '_');
+    const normalizedFilePath = context.task.name.replace(/([/: ])/g, '_').replace(/(['"><|*?\r\n])/g, '');
```

Comment:
I was wondering if we should not like replace all non letters/digits characters ?

something like `replace(/[^a-zA-Z0-9]/g, '')`;

Reply:
Seems more readable

Reply:
I have included a bit more sophisticated replacement that will remove most characters that are not letters/numbers or underscore, which I want to use as a delimiter to a words in the filename to keep better readability.